packages/tests/server/TRPCError.test.ts

```diff
@@ -1,77 +1,75 @@
 import { TRPCError, getTRPCErrorFromUnknown } from '@trpc/server/src';
 
-describe('TRPCError', () => {
-  it('should extend original Error class', () => {
-    const trpcError = new TRPCError({ code: 'FORBIDDEN' });
-    expect(trpcError).toBeInstanceOf(TRPCError);
-    expect(trpcError).toBeInstanceOf(Error);
-  });
+test('should extend original Error class', () => {
```

Comment:
There is more than 1 top-level thing tested in this file, that's why it was split using `describe` blocks :)